Result/Result.swift

```diff
@@ -24,11 +24,18 @@ public enum Result<T, Error: Swift.Error>: ResultProtocol, CustomStringConvertib
 
 	/// Constructs a result from a function that uses `throw`, failing with `Error` if throws.
 	public init(_ f: @autoclosure () throws -> T) {
-		self.init(attempt: f)
+		self.init(_attempt: f)
 	}
 
-	/// Constructs a result from a function that uses `throw`, failing with `Error` if throws.
+	@available(*, deprecated, renamed:"init(_:)")
 	public init(attempt f: () throws -> T) {
```

Comment:
This doesn't seem equivalent to `init(_:)` because the latter is in a constrained extension where `Error == AnyError`.

So before you could write `Result<Int, NSError>(attempt: f)`, but now there's no equivalent.